config/config.exs

```diff
@@ -35,8 +35,13 @@ config :lightning, Lightning.Extensions,
   usage_limiter: Lightning.Extensions.UsageLimiter,
   run_queue: Lightning.Extensions.FifoRunQueue
 
+# TODO: don't use this value in production
```

Comment:
what does this todo mean and when must it be done?

Reply:
Ah yes thanks for spotting this; will message you about it.

Reply:
I've made an issue over here: https://github.com/OpenFn/lightning/issues/1911